templates/analytics/realm_summary_table.html

```diff
@@ -3,9 +3,12 @@ <h2 style="text-align: center;">{{ num_active_sites }} active sites</h2>
 <p id="utctime" style="text-align: center;"></p>
 <p id="localtime" style="text-align: center;"></p>
 <script>
-    var now = moment('{{ now }}');
-    $('#utctime')[0].innerHTML = moment.utc(now).format('YYYY-MM-DD HH:mm') + 'Z';
-    $('#localtime')[0].innerHTML = '(' + now.format('YYYY-MM-DD HH:mm ZZ') + ')';
+    const {parseISO, format} = require("date-fns");
```

Comment:
`require` does not work in an inline `<script>`. We’ll need to migrate this template to properly use Webpack.

Reply:
Hey , Would be great if you could provide more information about this .

Reply:
I removed this `<script>` in #16501.